tools/tf2openapi/types/tftensor.go

```diff
@@ -111,6 +111,54 @@ func NewTFDType(name string, dType string) (TFDType, error) {
 	return tfDType, nil
 }
 
-func (t *TFTensor) Schema() *openapi3.Schema {
-	return &openapi3.Schema{}
+func (t *TFTensor) RowSchema() *openapi3.Schema {
+	// ignore the 0th dimension because it is always -1 in row fmt to represent batchable input
```

Comment:
Is it safe to guarantee that `schema(...)` method won't recurse forever? Otherwise it's better to have a sanity check   here.